cloudevents/spec.md

```diff
@@ -153,7 +153,9 @@ together into a transport envelope body.
 
 Every CloudEvent conforming to this specification MUST include context
 attributes designated as REQUIRED, MAY include one or more OPTIONAL context
-attributes and MAY include one or more extension attributes.
+attributes and MAY include one or more
+[extension context attributes](#extension-context-attributes). Each
+context attribute MUST only appear at most once in a CloudEvent.
```

Comment:
FYI - this new sentence at the end is the real change. The other stuff in this PR is either reformatting or s/field/attribute/ for consistency.